### PR TITLE
Adds the AGLS-37

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
Adds the AGLS-37 mounted automatic grenade launching system, which launches HE and Fragmentation shells for 100 points.

I stole the sprites from Tye, other than the ammo which was me.
image

Gun itself
-
It has no swivels, it cannot be turned without grabbing it and redeploying it.
It cannot move on its own.
~~It has a 0.75 fire rate. Could be tuned further.~~ Toned to 1.2
30+1 grenades in a drum.
It is not free roundstart, it costs 100 points, 10 per drum of ammo.
It uses the TL-102 scope wholesale.

Ammo
-
HE is just HEDPs with lower det time, they fly until they hit a surface or cross 21 tiles then detonate shortly after.
Frag fires a shell which flies 21 tiles or hits something then splits into 20 fragments, like the TX-54 but it has no mob passing on the shrapnel.
Shrapnel does 25/20/1, 360 cone. 20 shrapnel.

## Why It's Good For The Game
Adds a another unique choice to the mounted weapon category and uses an unused mechanic of grenade canisters for the ability to fire grenades without them instantly exploding in your face, and thus balancing this probably broken thing otherwise.

Fragmentation could be subject to change, but I like it as a mechanic, and the gun itself requires 2 people to effectively use without blowing yourself up due to the short range of the scope.

## Changelog
:cl:
add: Added new automated grenade launcher system for 100 points in req, go buy it. Its cool, fires impact shrapnel shell in a 360 cone like TX-54, and delayed explosive shells equivalent to HEDP.
/:cl:

### CODE REVIEW

File: code/modules/projectiles/guns/mounted.dm [hunk 1]
```diff
@@ -210,6 +210,47 @@
 	undeploy_time = 0.5 SECONDS
 	max_integrity = 200
 
+
+//AGLS-37, or Automatic Grenade Launching System 37, a fully automatic mounted grenade launcher that fires fragmentation and HE shells, can't be turned.
+
+/obj/item/weapon/gun/standard_agls
+	name = "\improper AGLS-37 Kauser automatic grenade launcher"
+	desc = "The AGLS-37 automatic grenade launching system, it's too heavy to be wielded or operated without the tripod. IFF capable. On the back, it reads 'The Explosions and Fragmentation from this weapon ARE NOT friendly fire capable. Kauser is not obligated to buy you new bodyparts for you or your friends if you lose them.'  Can be Deployed with Crtl-Click.   It CANNOT be turned once deployed, due to a lack of swivels, pick it up to move your cone of fire. Can be repaired with a blowtorch once deployed."
```
Comment (XSlayer300) [inline code/modules/projectiles/guns/mounted.dm:218]:
```suggestion
	desc = "The AGLS-37 automatic grenade launching IFF capable system, it's too heavy to be wielded or operated without the tripod. On the back, it reads: \"The Explosions and Fragmentation from this weapon ARE NOT friendly fire capable. Kauser is not obligated to buy you new body parts for you or your friends if you lose them.\"\nCan be deployed with Crtl-Click. It CANNOT be turned once deployed, due to a lack of swivels, pick it up to move your cone of fire. Can be repaired with a blowtorch once deployed."
```

**Comment by lKiseki** [comment]
can someone stop mjp from adding more deployables please thanks

**Comment by goose4429** [comment]
>> It is not free roundstart

Finally.

**Comment by Lumipharon** [comment]
With that fire rate it's gg to any xeno caught in the open, as you'll just be juggled to death without being able to do anything.

File: code/modules/projectiles/guns/mounted.dm [hunk 2]
```diff
@@ -210,6 +210,47 @@
...
+	desc = "The AGLS-37 automatic grenade launching IFF capable system, it's too heavy to be wielded or operated without the tripod. On the back, it reads: \"The Explosions and Fragmentation from this weapon ARE NOT friendly fire capable. Kauser is not obligated to buy you new body parts for you or your friends if you lose them.\"\nCan be deployed with Crtl-Click. It CANNOT be turned once deployed, due to a lack of swivels, pick it up to move your cone of fire. Can be repaired with a blowtorch once deployed."
+	w_class = WEIGHT_CLASS_HUGE
+	flags_equip_slot = ITEM_SLOT_BACK
+	icon = 'icons/Marine/marine-hmg.dmi'
+	icon_state = "ags"
+	fire_sound = 'sound/weapons/guns/fire/ags.ogg'
+	reload_sound = 'sound/weapons/guns/interact/minigun_cocked.ogg'
+	default_ammo_type = /obj/item/ammo_magazine/standard_agls
+	scatter = 0
+	fire_delay = 0.75 SECONDS
```
Comment (novaepee) [inline code/modules/projectiles/guns/mounted.dm:227]: Oh, lord. Lumi is right. 0.75 seconds? That's way faster than GL.
image
image

**Review by novaepee — CHANGES_REQUESTED** [review]
Change AGLS fire delay to equal or greater than the grenade launchers. Less than is bad for xenomorphs.

**Review by Kuro020 — APPROVED** [review]

**Comment by Pariah919** [comment]
will wait for lootboxes™️

**Pull request closed without merging**